pkg/message/slack.go

```diff
@@ -44,6 +48,40 @@ func (s *Slack) SendPullRequestEvent(URL, title, author string) error {
 	return s.sendMessage(asJSON)
 }
 
+func (s *Slack) NotifyMergeRequestCreated(mergeRequest models.MergeRequestInfo) error {
+	for _, channel := range s.ChannelsForMergeRequest(mergeRequest) {
+		var m = message{
+			Text: fmt.Sprintf("%s <%s|%s> by %s", s.message, mergeRequest.ObjectAttributes.URL, mergeRequest.ObjectAttributes.Title, mergeRequest.User.Name),
+			IconURL: s.avatar,
+			Username: s.username,
+			Channel: channel,
+		}
+
+		asJSON, err := json.Marshal(m)
+		if err != nil {
+			return err
+		}
+
+		return s.sendMessage(asJSON)
+	}
+
+	return nil
+}
+
+func (s *Slack) ChannelsForMergeRequest(mergeRequest models.MergeRequestInfo) []string {
+	channels := []string{};
+
+	for _, config := range s.notifications {
```

Comment:
Having `s.notifications` as `map[string][]string` we can:
```
for _, mrLabel := range mergeRequest.Labels {
    if c, ok := s.notifications; ok {
        channels = append(channels, c)
    }
}
```

Reply:
❤️